.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "tuesday"
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "tuesday"
```

Comment:
Please add a trailing blank line

Reply:
alright!

Reply:
Please add in `.github/workflows/go.yml` as well.

Reply:
I added the go.yml.

Reply:
like in dependabot or the file  `go.yml`